docs/docs/deployment/post_deployment/aws_accounts.md

```diff
@@ -66,104 +66,135 @@ To create the account, perform the following actions:
 
-_**Table 4: Stack Output**_
+_**Figure 5: Displaying the CloudFormation stack name**_
 
-6. Copy the values down for the next step.
+6. Choose **Create Stack**.
+7. The **Quick create stack** page appears and it displays the template URL, stack name and parameters. 
 
-### Adding the Account in Service Workbench
+<img src={useBaseUrl('img/deployment/post_deployment/quick-create-stack.png')} />
 
-This step is run in the Service Workbench administrator interface and uses values from the previous step.
+_**Figure 6: Creating the stack**_
 
-1. In the Service Workbench administrative interface, click the **AWS Accounts** tab. See **Figure 3**.
+8. Select **I acknowledge that AWS CloudFormation might create IAM resources with custom names**.
 
-<img src={useBaseUrl('img/deployment/post_deployment/create_account_01.jpg')} />
+<img src={useBaseUrl('img/deployment/post_deployment/acknowledge.png')} />
 
-_**Figure 3: Add AWS Account**_
+_**Figure 7: Acknowlegement screen**_
 
-2.  Click **Add AWS Account**. Enter the account information from the following **Table 5**:
+9. Choose **Create stack**.
 
-|             Field            |                 Value                  |
-|------------------------------|----------------------------------------|
-| Account Name                 | As desired                             |
-| AWS Account ID               | 12-digit ID of imported account        |
-| Role ARN                     | **CrossAccountExecutionRoleArn** value |
-| AWS Service Catalog Role Arn | **CrossAccountEnvMgmtRoleArn** value   |
-| External ID                  | As specified (default: **workbench**)  |
-| Description                  | As desired                             |
-| VPC ID                       | **VPC** value                          |
-| Subnet ID                    | **VpcPublicSubnet1** value             |
-| KMS Encryption Key ARN       | **EncryptionKeyArn** value             |
+10. Once the account is added it will be listed in AWS Accounts. When the associated cloudformation stack finishes provisioning, the account displays as **Up-to-Date**.
 
-_**Table 5: AWS Account Information**_
+<img src={useBaseUrl('img/deployment/post_deployment/new_account1.jpg')} />
 
-3. Once the account is added it will be listed in **AWS Accounts**, see **Figure 4**.
+_**Figure 8: AWS accounts with new account**_
+### Updating a previously onboarded account
 
-<img src={useBaseUrl('img/deployment/post_deployment/create_account_02.jpg')} />
+When new versions of Service Workbench are launched, it might be necessary to change the resources Service Workbench uses to access onboarded accounts. The AWS Accounts page displays information on which accounts are up-to-date, and which need to be updated. 
+
+For accounts that need to be updated, follow these steps:
+
+1. Sign in to the AWS Management Console for the corresponding account in a separate tab.
+2. On the **AWS Accounts** tab, choose **Update Permissions**.
```

Comment:
On the **AWS Accounts** tab, click on **Update Permissions** for the Account you would like to update.

Reply:
Its updated. Thanks